lua/go/tags.lua

```diff
@@ -34,17 +36,27 @@ tags.modify = function(...)
   end
   local arg = { ... }
   local transflg = false
+  local optsflg = false
   for _, v in ipairs(arg) do
     table.insert(setup, v)
     if v == "-transform" or v == "-t" then
       transflg = true
     end
+    if v == "-add-options" then
+      optsflg = true
```

Comment:
How about allowing the user to pass in the options here 
e.g. 
`GoModifyTags -add-option json=omitempty`